tiledb/sm/array_schema/attribute.cc

```diff
@@ -442,6 +420,22 @@ std::optional<std::string> Attribute::get_enumeration_name() const {
   return enumeration_name_;
 }
 
+std::string Attribute::fill_value_str() const {
```

Comment:
Now that this function is feeding an output stream, it would probably be better to inline its body directly into its call site in `operator<<`. It was always called in only one place.

Alternately, the only member variable used is `fill_value_`, which is available publicly as `fill_value()`, so this function could simply be defined locally, appearing in the `.cc` file immediately prior to the implementation of `operator<<`.

Regardless, there's no need for it in the public interface of this class.